src/mocks/msw/handlers.ts

```diff
@@ -178,14 +178,18 @@ const handlers = [
     )}`,
     async (req, res, ctx) => {
       let status = 404
+      const entityId = req.params.entityId
+      const versionNumber = req.params.versionNumber.toString()
```

Comment:
MSW type changed req.params.versionNumber to be `string | readonly string` which causes some annoying (but not build-breaking since this is only exposed in test) type errors. This fixes it.